os/dbuild.sh

```diff
@@ -103,11 +103,11 @@ function SELECT_OPTION()
 			echo ======================================================
 			echo "  \"Select build Option\""
 			echo ======================================================
-			echo "  \"1. Build with Current Configurations\""
+			echo "  \"1. Build with Current Configuration\""
 			echo "  \"2. Re-configure\""
-			echo "  \"3. Menuconfig\""
-			echo "  \"4. Build Clean\""
-			echo "  \"5. Build Dist-Clean\""
+			echo "  \"3. Modify Current Configuration\""
+			echo "  \"4. Clean Build\""
```

Comment:
1. How about this? `Clean build`? `Clean output`?
2. for 5, Re-Configuration -> Re-configure

Reply:
1. Lets keep it as `Clean build` 
2. `Re-configure` sounds good

Reply:
ok

Reply:
Could you update the commit description with new sentence?

Reply:
done